### PR TITLE
4.x - fix for botocore 1.33.2 (based off Pull Request #1206 made for 5.x branch)

### PR DESCRIPTION
**Pull request by canopus1io**
@ikonst can you please review this pull request based on your pull request #1206 to make sure I didn't miss anything

let me know if there is anything more I need to do to get this merged and released

### CODE REVIEW

**Comment by Yaronn44** [comment]
Can someone check this PR and merge it please ? I still have some old server's versions that rely on the 4.x.x version of PynamoDB 
This would be greatly appreciated

File: docs/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 .[signals]
-sphinx-rtd-theme==0.4.3
+sphinx>=5
+sphinx-rtd-theme==1.1.1
+sphinx-issues
```
Comment (ikonst) [inline docs/requirements.txt:4]:
```suggestion
sphinx-issues
```

**Comment by ikonst** [comment]
I'm running into trouble making a release.

**Comment by Yaronn44** [comment]
Just tried to use this new version as I received the notification for the new release and just saw that it does not exist. I'll wait that you resolve the problem then. Thank you very much for being so responsive !

**Comment by ikonst** [comment]
released! - https://pypi.org/project/pynamodb/4.4.1/

**Comment by canopus1io** [comment]
> released! - https://pypi.org/project/pynamodb/4.4.1/

thank you, I appreciate you getting this merged and released

**Pull request merged**